api/src/backend/api/migrations/0010_scan_report_output.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.1.5 on 2025-02-07 10:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("api", "0009_increase_provider_uid_maximum_length"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="scan",
+            name="output_path",
```

Comment:
```suggestion
            name="output_location",
```

What about `output_location`? I think it'll be more generic for future approaches, like S3 now but other file locations in the future. I'd also like to define if we are going to call this `reports` or `outputs`.